src/Numerics/Optimizers/ExhaustiveOptimizer/CMakeLists.txt

```diff
@@ -14,13 +14,25 @@ install(TARGETS ExhaustiveOptimizer
   COMPONENT Runtime
   )
 
-install(FILES Code.cxx CMakeLists.txt
+install(FILES Code.cxx exhaustiveoptimizer.py CMakeLists.txt
```

Comment:
Following the file naming convention, the name should be *Code.py*.

Reply:
This change was originally intended to allow the accompanying notebook to import functions from the example as part of a module, noting that `import code` fails likely due to collision with the built-in [code](https://docs.python.org/3/library/code.html) module. After splitting the notebook into [PR 194](https://github.com/InsightSoftwareConsortium/ITKExamples/pull/194) its dependency on the example has been removed, so reverting to the naming convention `code.py` is an easy fix. In the future there may be instances where the collision comes into consideration.